iot_input/views/iot_device_input_views.xml

```diff
@@ -66,7 +66,6 @@
         <field name="name">IoT Inputs</field>
         <field name="type">ir.actions.act_window</field>
         <field name="res_model">iot.device.input</field>
-        <field name="view_type">form</field>
```

Comment:
In v12 you still have this. Is in v13 that got deleted.

Reply:
Done!